### ISSUE TITLE
autotools checks in mathicgb, memtailor, mathic fail

### ISSUE THREAD

**Issue by mikestillman** [Macaulay2/M2#2479]
On a Mac M1, I get the following error during a `gmake -k check` (in the Macaulay2 build).  @DanGrayson @d-torrance  Is it easy to change the tests to not change the ulimit on the mac?

```
gmake: *** [GNUmakefile:251: check-in-Macaulay2] Error 2
cd ../../../../.. && git submodule update --init M2/submodules/memtailor
ulimit -v 1000000 && gmake -C submodules/memtailor check
/bin/sh: line 0: ulimit: virtual memory: cannot modify limit: Invalid argument
gmake: *** [GNUmakefile:210: check-in-memtailor] Error 1
cd ../../../../.. && git submodule update --init M2/submodules/mathic
ulimit -v 1000000 && gmake -C submodules/mathic check
/bin/sh: line 0: ulimit: virtual memory: cannot modify limit: Invalid argument
gmake: *** [GNUmakefile:210: check-in-mathic] Error 1
cd ../../../../.. && git submodule update --init M2/submodules/mathicgb
ulimit -v 1000000 && gmake -C submodules/mathicgb check
/bin/sh: line 0: ulimit: virtual memory: cannot modify limit: Invalid argument
gmake: *** [GNUmakefile:210: check-in-mathicgb] Error 1
```

**Comment by DanGrayson** [comment]
I get the same on the Mac book.  I'll fix it.

**Comment by DanGrayson** [comment]
fixed in commit 0bdc6c13013171a329e7d14f002c6c9d63aa3f46
